src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -772,8 +796,8 @@ def load_command_table(self, _):
                             managed_database_long_term_retention_policies_operations,
                             client_factory=get_sql_managed_database_long_term_retention_policies_operations) as g:
 
-        g.custom_command('set', 'update_long_term_retention_mi', is_preview=True)
-        g.show_command('show', 'get', is_preview=True)
+        g.custom_show_command('set', 'update_long_term_retention_mi', is_preview=True)
```

Comment:
Use `g.custom_command(`set`, xxx)` for `set` command as `custom_show_command` is only for `show` command`.

Reply:
addressed